IntegrationTests.Common/TestDomain/Northwind/NorthwindDataContext.cs

```diff
@@ -35,6 +35,12 @@
 using System.Linq;
 using System.Reflection;
 
+#if NET6_0_OR_GREATER
+using DateOnly2 = System.DateOnly;
+#else
+using DateOnly2 = System.DateTime;
+#endif
```

Comment:
Great this works. just for completeness' sake: if you got any good naming ideas instead of `DateOnly2` I'm happy to hear them. I just had no short name ideas myself.

Reply:
I think `LocalDateOnly` would fit.